frontend/src/main/scala/bloop/engine/SemanticDBCache.scala

```diff
@@ -0,0 +1,59 @@
+package bloop.engine
```

Comment:
@jvican  Still need to tidy up the PR and fix one test, but what do you think about `SemanticDBCache`? Is this something you had in mind?